fab/operations/offline.py

```diff
@@ -0,0 +1,54 @@
+import os
+
+from fabric.api import env, local
+from fab.utils import generate_bower_command
+from fab.const import (
+    OFFLINE_STAGING_DIR,
+    WHEELS_ZIP_NAME,
+    HQ_ZIP_NAME,
+    BOWER_ZIP_NAME,
+    NPM_ZIP_NAME,
+)
+
+
+def prepare_zipfiles():
+    local(
+        'git clone --depth 1 --recursive https://github.com/dimagi/commcare-hq.git {}/commcare-hq'
+        .format(OFFLINE_STAGING_DIR)
+    )
+
+    # After we've created our HQ code zip, let's get bower and npm
```

Comment:
I thought i made this comment already but don't see it here. Another possibility that might be easier is to just setup a git server on one of the machines and push/pull to it from behind the VPN. I've done this on some personal projects and it's not very hard (though might be hard if we want to do it via ansbile... not sure if we feel that would be necessary)